lib/game.rb

```diff
@@ -0,0 +1,67 @@
+require_relative './score'
+
+class Game
+  attr_reader :throws, :score, :frame_count, :last_frame
+
+  MAX_FRAME = 10
+
+  def initialize(score = Score.new)
+    @throws = []
+    @frame_complete = true
+    @score = score
+    @frame_count = 1
+    @last_frame = []
+  end
+
+  def add(pins)
+    @throws << pins
+    play_last_frame(pins) if last_frame?
+    frame_status(pins)
+    setup_next_roll(pins)
+  end
+
+  def frame_status(pins)
+    @frame_complete = !@frame_complete if pins != 10
```

Comment:
a bit opaque! maybe use the `frame_complete?` method since you have it? That would show that booleans are involved